toolset/setup/linux/prerequisites.sh

```diff
@@ -63,10 +63,16 @@ sudo pip install colorama==0.3.1
 sudo pip install progressbar==2.2
 sudo pip install requests
 
-# Install gcc-4.8, gcc-4.9 and clang
+# Install gcc-4.8, gcc-4.9
 sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
 sudo apt-get -yq update
-sudo apt-get install -qqy gcc-4.8 g++-4.8 gcc-4.9 g++-4.9 clang-3.5
+sudo apt-get install -qqy gcc-4.8 g++-4.8 gcc-4.9 g++-4.9
+
+# Install Clang, needed by silicon.
```

Comment:
That's alright.

I agree with @nbrady-techempower that we need to get as much of this out of the prerequisites as possible.